### ISSUE TITLE
Why fit_gpytorch_model does not use new data in different iterations?

### ISSUE THREAD

**Issue by machanic** [satyanshukla/bayes_attack#2]
In each iteration, the same `mll` is used in `fit_gpytorch_model(mll)`, which means the `new_x` and `new_obj` is not used.
It is located in https://github.com/satyanshukla/bayes_attack/blob/main/attack.py#L131
Why?

**Comment by machanic** [comment]
Is it a bug?

**Comment by satyanshukla** [comment]
No, if you see line 167, we set the training data for the model which incorporates the new data points.
